src/containers/Fleets/components/FleetsContent.js

```diff
@@ -122,7 +120,7 @@ class FleetsContent extends PureComponent {
         ...prevState,
         isLoading: true,
         itemID: getID(nextProps.history.location.pathname),
-        input: nextProps.selectedItems.length === 1 ? nextProps.selectedItems[0]['PluginFlyvemdmFleet.name'] : 'New Fleet',
+        input: nextProps.selectedItems.length === 1 ? nextProps.selectedItems[0]['PluginFlyvemdmFleet.name'] : 'New Feet',
```

Comment:
Review this 'New Feet'

Reply:
done! this string was added to translations file

Reply:
https://github.com/flyve-mdm/web-mdm-dashboard/pull/932/commits/2fd265fd3fb99095205682862d1a53df05dcabd2